service/service.go

```diff
@@ -87,6 +95,87 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	var newOperatorBackOff *backoff.ExponentialBackOff
+	{
+		newOperatorBackOff = backoff.NewExponentialBackOff()
+		newOperatorBackOff.MaxElapsedTime = config.Viper.GetDuration(config.Flag.Service.Operator.OperatorBackOffDuration)
+	}
+
+	var newServiceResource framework.Resource
```

Comment:
The naming seems odd to me, I am unsure if there needs to be a resource for `service`.

Reply:
See my comment above. I think it should be a endpoint resource.